crates/collab/src/rpc.rs

```diff
@@ -4462,26 +4462,29 @@ async fn complete_with_open_ai(
                         tool_calls: choice
                             .delta
                             .tool_calls
-                            .into_iter()
```

Comment:
I think we could `.unwrap_or_default()` first, and then we don't need the additional `.map` for the `Option`.